features/steps/core.py

```diff
@@ -119,24 +127,26 @@ def run_with_input(context, command, inputs=""):
         text = iter([inputs])
 
     args = ushlex(command)[1:]
+    # fmt: off
```

Comment:
Why are we turning it off for this file?

Reply:
Because otherwise the next three lines become rather mangled and less readable. I.e. this:

```
with patch("builtins.input", side_effect=_mock_input(text)) as mock_input,\
     patch("getpass.getpass", side_effect=_mock_getpass(text)) as mock_getpass,\
     patch("sys.stdin.read", side_effect=text) as mock_read:
```

becomes this:

```
with patch("builtins.input", side_effect=_mock_input(text)) as mock_input, patch(
    "getpass.getpass", side_effect=_mock_getpass(text)
) as mock_getpass, patch("sys.stdin.read", side_effect=text) as mock_read:
```

To get `black` to leave the context manager alone (i.e. the `with` statement), the whole block has to be turned off as `black` requires it's on/off to be at the same indentation level.

Reply:
Ugh, yeah, that's ugly. Looks like it might be a while for that to get fixed, too (https://github.com/psf/black/issues/557).  :(